### ISSUE TITLE
Create cookiecutter

### ISSUE THREAD

**Issue by stevepiercy** [plone/plone-sphinx-theme#3]
Create a cookiecutter `cookie-pst` that generates a documentation environment using this theme.

Write documentation of the cookiecutter.

Update this project's documentation with usage.

Template goes into https://github.com/plone/cookieplone-templates

See https://github.com/plone/cookieplone-templates/issues/7

**Comment by ujsquared** [comment]
hey! i have started working on this, are there any new requirements that the cookiecutter template may require?

**Comment by stevepiercy** [comment]
@ujsquared thanks for taking on this challenge. I can't think think of any new requirements since I created this issue.

The first requirement, "generates a documentation environment", is deliberately vague because I was not sure of details and I assumed I would figure it out as I worked through it. For example, I know that you should be able to build documentation from the root of the generated project with `make html` or `make livehtml`, most of which can be reused from what exists in this repo.

The PR, though, should be made in https://github.com/plone/cookieplone-templates. Feel free to ping me in Discord in the #Documentation channel if you have any questions.

**Comment by ujsquared** [comment]
@stevepiercy 

From what I've gathered so far of this issue, the goal of the proposed template is to minimize the effort and time spent on future documentation that will be created from scratch, whether internally at Plone or elsewhere. Is this correct?

**Comment by stevepiercy** [comment]
@ujsquared correct. In addition, it is a friendly reminder that "If it ain't documented, it's broken." This template would remove barriers to (and excuses from) developers writing documentation of their add-on or project.
